src/connector.js

```diff
@@ -480,7 +490,7 @@ function initEngine() {
 	statusController = buildSearchStatus( headlessEngine );
 
 	if ( urlParams.allq || urlParams.exctq || urlParams.anyq || urlParams.noneq || urlParams.fqupdate || 
-		urlParams.dmn || urlParams.fqocct || urlParams.elctn_cat || urlParams.filetype || urlParams.site || urlParams.year ) { 
```

Comment:
I don't think you should be removing the year feature since it is used for Budget Canada.